cmd/juju/application/deployer/bundle.go

```diff
@@ -213,7 +226,8 @@ func (d *deployBundle) makeBundleDeploySpec(ctx *cmd.Context, apiRoot DeployerAP
 		targetModelUUID:      d.targetModelUUID,
 		controllerName:       d.controllerName,
 		accountUser:          d.accountUser,
-	}
+		knownSpaces:          knownSpaces,
```

Comment:
I like `knownSpaceNames` for this.

Reply:
I used knownSpaces because that's consistent with what is used here and the like juju/cmd/juju/application/binding.go

But you're right, knownSpaceNames works better. As a flyby I'll change both sets to that